src/components/bulk-email-tool/bulk-email-task-manager/BulkEmailContentHistory.jsx

```diff
@@ -1,8 +1,23 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
 
 import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { getSentEmailHistory } from './api';
 
 export default function BulkEmailContentHistory() {
+  const { courseId } = useParams();
+
+  const [emailHistoryData, setEmailHistoryData] = useState(); // eslint-disable-line no-unused-vars
```

Comment:
While `emailHistoryData` isn't being used _now_, we will use it in a follow-up to this work. Disabling the es-lint check is just temporary and will be removed.